{"context": "\n\nEl captain\u306epython\u30c7\u30a3\u30ec\u30af\u30c8\u30ea\u304c/usr/bin\u3060\u304b\u3089\u3061\u304f\u3057\u3087\u3046!!(\u5f7c\u5cb8\u5cf6)\nMac\u306eOS\u3092El captain\u306b\u3057\u305f\u5f8c\u306bJupyter\u3084Tensorflow\u306a\u3069pip\u3067\u30a4\u30f3\u30b9\u30c8\u30fc\u30eb\u3057\u3088\u3046\u3068\u3057\u305f\u3089\u898b\u305f\u3053\u3068\u304c\u306a\u3044\u30a8\u30e9\u30fc\u304c\u591a\u767a\u3057\u307e\u3057\u305f\u3002\n% pip install https://storage.googleapis.com/tensorflow/mac/tensorflow-0.5.0-py2-none-any.whl                                    \nCollecting tensorflow==0.5.0 from https://storage.googleapis.com/tensorflow/mac/tensorflow-0.5.0-py2-none-any.whl\n  Using cached https://storage.googleapis.com/tensorflow/mac/tensorflow-0.5.0-py2-none-any.whl\nCollecting six>=1.10.0 (from tensorflow==0.5.0)\n  Using cached six-1.10.0-py2.py3-none-any.whl\nCollecting numpy>=1.9.2 (from tensorflow==0.5.0)\n  Using cached numpy-1.10.1-cp27-none-macosx_10_6_intel.macosx_10_9_intel.macosx_10_9_x86_64.macosx_10_10_intel.macosx_10_10_x86_64.whl\nInstalling collected packages: six, numpy, tensorflow\n  Found existing installation: six 1.4.1\n    DEPRECATION: Uninstalling a distutils installed project (six) has been deprecated and will be removed in a future version. This is due to the fact that uninstalling a distutils project will only partially uninstall the project.\n    Uninstalling six-1.4.1:\nException:\nTraceback (most recent call last):\n  File \"/Library/Python/2.7/site-packages/pip-7.1.2-py2.7.egg/pip/basecommand.py\", line 211, in main\n    status = self.run(options, args)\n  File \"/Library/Python/2.7/site-packages/pip-7.1.2-py2.7.egg/pip/commands/install.py\", line 311, in run\n    root=options.root_path,\n  File \"/Library/Python/2.7/site-packages/pip-7.1.2-py2.7.egg/pip/req/req_set.py\", line 640, in install\n    requirement.uninstall(auto_confirm=True)\n  File \"/Library/Python/2.7/site-packages/pip-7.1.2-py2.7.egg/pip/req/req_install.py\", line 716, in uninstall\n    paths_to_remove.remove(auto_confirm)\n  File \"/Library/Python/2.7/site-packages/pip-7.1.2-py2.7.egg/pip/req/req_uninstall.py\", line 125, in remove\n    renames(path, new_path)\n  File \"/Library/Python/2.7/site-packages/pip-7.1.2-py2.7.egg/pip/utils/__init__.py\", line 315, in renames\n    shutil.move(old, new)\n  File \"/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/shutil.py\", line 302, in move\n    copy2(src, real_dst)\n  File \"/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/shutil.py\", line 131, in copy2\n    copystat(src, dst)\n  File \"/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/shutil.py\", line 103, in copystat\n    os.chflags(dst, st.st_flags)\nOSError: [Errno 1] Operation not permitted: '/var/folders/98/f5yhj5k51r18qfgxm8xx83v40000gn/T/pip-s4Fqnv-uninstall/System/Library/Frameworks/Python.framework/Versions/2.7/Extras/lib/python/six-1.4.1-py2.7.egg-info'\n\nwhich python\u3057\u3066\u307f\u305f\u3089\u3001OS\u306b\u4ed8\u5c5e\u306epython\u304c/usr/bin/\u76f4\u4e0b\u306b\u3042\u308a\u3053\u308c\u304c\u539f\u56e0\u306e\u69d8\u5b50\u3002python\u3092brew\u3067\u518d\u30a4\u30f3\u30b9\u30c8\u30fc\u30eb\u3057\u305f\u3089python\u304c/usr/local/bin\u306b\u304a\u3055\u307e\u308a\u3001\u305d\u306e\u5f8c\u306e\u30a4\u30f3\u30b9\u30c8\u30fc\u30eb\u306f\u3059\u3079\u3066\u3046\u307e\u304f\u3044\u304d\u307e\u3057\u305f\u3002\nbrew update\nbrew install python\nwhich python\n/usr/local/bin/python\n\n\n\u5f8c\u65e5\u8ac7\n\u305d\u306e\u5f8cR\u3067ggplot2\u3084rjags\u3092\u5165\u308c\u3088\u3046\u3068\u3057\u305f\u3089\u4e0b\u8a18\u306e\u3088\u3046\u306a\u30a8\u30e9\u30fc\u3067\u306f\u307e\u308a\u307e\u3057\u305f\u3002\n\u8b66\u544a\u30e1\u30c3\u30bb\u30fc\u30b8: \n doTryCatch(return(expr), name, parentenv, handler) \u3067: \n   \u5171\u6709\u30e9\u30a4\u30d6\u30e9\u30ea '/Library/Frameworks/R.framework/Resources/modules//R_X11.so' \u3092\u8aad\u307f\u8fbc\u3081\u307e\u305b\u3093: \n  dlopen(/Library/Frameworks/R.framework/Resources/modules//R_X11.so, 6): Library not loaded: /opt/X11/lib/libSM.6.dylib\n  Referenced from: /Library/Frameworks/R.framework/Resources/modules//R_X11.so\n  Reason: image not found \n\u3000\u3000\uff1a\n\u3000Reason: no suitable image found.  Did find:\n    /opt/X11/lib/libSM.6.dylib: stat() failed with errno=62 \n\nhttp://stackoverflow.com/questions/26489928/cant-load-x11-in-r-after-os-x-yosemite-upgrade\n\n\u3053\u3061\u3089\u3092\u53c2\u8003\u306b\u4e0b\u8a18\u3092\u5b9f\u65bd\u3057\u305f\u3089\u306a\u304a\u308a\u307e\u3057\u305f\u3002\n\nsudo ln -s /usr/X11 /opt/X11 \nlocate libSM.6.dylib\n\n#El captain\u306epython\u30c7\u30a3\u30ec\u30af\u30c8\u30ea\u304c/usr/bin\u3060\u304b\u3089\u3061\u304f\u3057\u3087\u3046!!(\u5f7c\u5cb8\u5cf6)\nMac\u306eOS\u3092El captain\u306b\u3057\u305f\u5f8c\u306bJupyter\u3084Tensorflow\u306a\u3069pip\u3067\u30a4\u30f3\u30b9\u30c8\u30fc\u30eb\u3057\u3088\u3046\u3068\u3057\u305f\u3089\u898b\u305f\u3053\u3068\u304c\u306a\u3044\u30a8\u30e9\u30fc\u304c\u591a\u767a\u3057\u307e\u3057\u305f\u3002\n\n```\n% pip install https://storage.googleapis.com/tensorflow/mac/tensorflow-0.5.0-py2-none-any.whl                                    \nCollecting tensorflow==0.5.0 from https://storage.googleapis.com/tensorflow/mac/tensorflow-0.5.0-py2-none-any.whl\n  Using cached https://storage.googleapis.com/tensorflow/mac/tensorflow-0.5.0-py2-none-any.whl\nCollecting six>=1.10.0 (from tensorflow==0.5.0)\n  Using cached six-1.10.0-py2.py3-none-any.whl\nCollecting numpy>=1.9.2 (from tensorflow==0.5.0)\n  Using cached numpy-1.10.1-cp27-none-macosx_10_6_intel.macosx_10_9_intel.macosx_10_9_x86_64.macosx_10_10_intel.macosx_10_10_x86_64.whl\nInstalling collected packages: six, numpy, tensorflow\n  Found existing installation: six 1.4.1\n    DEPRECATION: Uninstalling a distutils installed project (six) has been deprecated and will be removed in a future version. This is due to the fact that uninstalling a distutils project will only partially uninstall the project.\n    Uninstalling six-1.4.1:\nException:\nTraceback (most recent call last):\n  File \"/Library/Python/2.7/site-packages/pip-7.1.2-py2.7.egg/pip/basecommand.py\", line 211, in main\n    status = self.run(options, args)\n  File \"/Library/Python/2.7/site-packages/pip-7.1.2-py2.7.egg/pip/commands/install.py\", line 311, in run\n    root=options.root_path,\n  File \"/Library/Python/2.7/site-packages/pip-7.1.2-py2.7.egg/pip/req/req_set.py\", line 640, in install\n    requirement.uninstall(auto_confirm=True)\n  File \"/Library/Python/2.7/site-packages/pip-7.1.2-py2.7.egg/pip/req/req_install.py\", line 716, in uninstall\n    paths_to_remove.remove(auto_confirm)\n  File \"/Library/Python/2.7/site-packages/pip-7.1.2-py2.7.egg/pip/req/req_uninstall.py\", line 125, in remove\n    renames(path, new_path)\n  File \"/Library/Python/2.7/site-packages/pip-7.1.2-py2.7.egg/pip/utils/__init__.py\", line 315, in renames\n    shutil.move(old, new)\n  File \"/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/shutil.py\", line 302, in move\n    copy2(src, real_dst)\n  File \"/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/shutil.py\", line 131, in copy2\n    copystat(src, dst)\n  File \"/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/shutil.py\", line 103, in copystat\n    os.chflags(dst, st.st_flags)\nOSError: [Errno 1] Operation not permitted: '/var/folders/98/f5yhj5k51r18qfgxm8xx83v40000gn/T/pip-s4Fqnv-uninstall/System/Library/Frameworks/Python.framework/Versions/2.7/Extras/lib/python/six-1.4.1-py2.7.egg-info'\n```\n\nwhich python\u3057\u3066\u307f\u305f\u3089\u3001OS\u306b\u4ed8\u5c5e\u306epython\u304c/usr/bin/\u76f4\u4e0b\u306b\u3042\u308a\u3053\u308c\u304c\u539f\u56e0\u306e\u69d8\u5b50\u3002python\u3092brew\u3067\u518d\u30a4\u30f3\u30b9\u30c8\u30fc\u30eb\u3057\u305f\u3089python\u304c/usr/local/bin\u306b\u304a\u3055\u307e\u308a\u3001\u305d\u306e\u5f8c\u306e\u30a4\u30f3\u30b9\u30c8\u30fc\u30eb\u306f\u3059\u3079\u3066\u3046\u307e\u304f\u3044\u304d\u307e\u3057\u305f\u3002\n\n```\nbrew update\nbrew install python\nwhich python\n/usr/local/bin/python\n```\n\n#\u5f8c\u65e5\u8ac7\n\u305d\u306e\u5f8cR\u3067ggplot2\u3084rjags\u3092\u5165\u308c\u3088\u3046\u3068\u3057\u305f\u3089\u4e0b\u8a18\u306e\u3088\u3046\u306a\u30a8\u30e9\u30fc\u3067\u306f\u307e\u308a\u307e\u3057\u305f\u3002\n\n```\n\u8b66\u544a\u30e1\u30c3\u30bb\u30fc\u30b8: \n doTryCatch(return(expr), name, parentenv, handler) \u3067: \n   \u5171\u6709\u30e9\u30a4\u30d6\u30e9\u30ea '/Library/Frameworks/R.framework/Resources/modules//R_X11.so' \u3092\u8aad\u307f\u8fbc\u3081\u307e\u305b\u3093: \n  dlopen(/Library/Frameworks/R.framework/Resources/modules//R_X11.so, 6): Library not loaded: /opt/X11/lib/libSM.6.dylib\n  Referenced from: /Library/Frameworks/R.framework/Resources/modules//R_X11.so\n  Reason: image not found \n\u3000\u3000\uff1a\n\u3000Reason: no suitable image found.  Did find:\n\t/opt/X11/lib/libSM.6.dylib: stat() failed with errno=62 \n\nhttp://stackoverflow.com/questions/26489928/cant-load-x11-in-r-after-os-x-yosemite-upgrade\n```\n\n[\u3053\u3061\u3089](http://stackoverflow.com/questions/26489928/cant-load-x11-in-r-after-os-x-yosemite-upgrade)\u3092\u53c2\u8003\u306b\u4e0b\u8a18\u3092\u5b9f\u65bd\u3057\u305f\u3089\u306a\u304a\u308a\u307e\u3057\u305f\u3002\n```\nsudo ln -s /usr/X11 /opt/X11 \nlocate libSM.6.dylib\n```\n", "tags": ["Python", "Mac", "Jupyter", "IPython", "pip"]}